submissions/TaniaBondarenko/frogger_game/app.js

```diff
@@ -0,0 +1,123 @@
+const borders = {
+    rightBorder: 400,
+    leftBorder: 5,
+    upBorder: 410,
+    downBorder: 5
+};
+const someGap = 10;
+
+class GameSprite {
+    constructor (x, y,sprite)  {
+        this.sprite = sprite;
+        this.x = x;
+        this.y = y;
+    }
+    render = function () {
+        ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+    };
+}
+
+class Enemy extends GameSprite {
+    constructor(x, y, sprite = "images/enemy-bug.png", speed = "75") {
+        super(x, y, sprite);
+        this.x = x;
+        this.y = y;
+        this.speed = speed;
+        this.sprite = sprite;
```

Comment:
this.x, 
this.y, 
this.sprite
  - are already defined in line 22 with the parent constructor that we call here:  super(x, y, sprite);
we don't need to redefine them again...

Reply:
Sure.